src/core/client/stream/mutations/SetCommentIDMutation.ts

```diff
@@ -19,6 +20,16 @@ export async function commit(
     const record = store.get(LOCAL_ID)!;
     record.setValue(input.id, "commentID");
     record.setValue("COMMENTS", "activeTab");
+
+    // Change iframe url, this is important
+    // because it is used to cleanly initialized
+    // a user session.
+    window.history.replaceState(
```

Comment:
Why are you using `replaceState` instead of `pushState`?

Reply:
Intentional, because `pushState` would change where the browser would go to when clicking `back`, but in this case we are in the iFrame and we don't want `back` to change anything inside of the iframe. The reason we are adapting the url is because, when a new user session is created, we freshly initialise our local state which looks into the url for e.g. the commentID.

In the `embed` we have the equivalent code that changes the history using `pushState`, in this case we want `back` to change the parent url.